karma.conf.ts

```diff
@@ -21,6 +23,13 @@ module.exports = (config) => {
                 flags: ["--no-sandbox"],
             },
         },
+        client: {
+            jasmine: {
+                random: false
+            },
+            // Make RUN_FAILING_TESTS available to browser tests
+            runFailingTests: runFailingTests
```

Comment:
There's some bugs that the tests have revealed (or at least I think they're bugs) - run with this flag to see them fail.